### PR TITLE
Add cessation date display to session content labels.

### PR DESCRIPTION
**Pull request by eschlange**
* Added integration points to session content.
* Added controller methods to transform session content.
* Added specs for new controller methods.

[#103907650]

### CODE REVIEW

File: test/unit/controllers/SessionsControllerSpec.js [hunk 1]
```diff
@@ -242,4 +251,20 @@ describe('SessionsController', function () {
...
+      var expectedResult = "this is a cessation date: Thursday, Jan. 1";
+      expect(controller.transformedIntegrationDisplay(rawContent)).to.eq(expectedResult);
+    });
+  });
+
+  describe('#formattedCessationDate', function () {
+    it('should retreive the cessation date and format it.', function () {
+      var expectedResult = "Thursday, Jan. 1";
+      expect(controller.formattedCessationDate()).to.eq(expectedResult);
+      expect(configurationServiceGetCessationDateSpy.calledOnce).to.be.true;
```
Comment (michaelwehrley) [inline test/unit/controllers/SessionsControllerSpec.js]: Do you need to check this if the result is correct?

**Comment by michaelwehrley** [comment]
:+1: looks good to me!

File: test/unit/controllers/SessionsControllerSpec.js (hunk 1 above)
Comment (eschlange) [inline test/unit/controllers/SessionsControllerSpec.js]: Yes.

**Comment by michaelwehrley** [comment]
:eyes: :two_men_holding_hands: while @eschlange and I argue about :spies!

**Pull request merged**